From: Markus Koschany <apo@debian.org>
Date: Tue, 14 Aug 2018 12:42:21 +0200
Subject: 01-use-jar-files-from-debian

Use existing jar files during the build instead of
downloading the dependencies.

Forwarded: not-needed
---
 src/tools/org/h2/build/Build.java | 13 ++++++-------
 1 file changed, 6 insertions(+), 7 deletions(-)

--- a/src/tools/org/h2/build/Build.java
+++ b/src/tools/org/h2/build/Build.java
@@ -285,14 +285,14 @@
         switchSource(debugInfo);
         clean();
         mkdir("temp");
-        download();
         String classpath = "temp" +
-                File.pathSeparator + "ext/servlet-api-3.1.0.jar" +
-                File.pathSeparator + "ext/lucene-core-3.6.2.jar" +
-                File.pathSeparator + "ext/slf4j-api-1.6.0.jar" +
-                File.pathSeparator + "ext/org.osgi.core-4.2.0.jar" +
-                File.pathSeparator + "ext/org.osgi.enterprise-4.2.0.jar" +
-                File.pathSeparator + "ext/jts-core-1.15.0.jar" +
+                File.pathSeparator + "/usr/share/java/servlet-api-3.1.jar" +
+                File.pathSeparator + "/usr/share/java/lucene-analyzers-common-4.10.4.jar" +
+                File.pathSeparator + "/usr/share/java/lucene-core-4.10.4.jar" +
+                File.pathSeparator + "/usr/share/java/lucene-queryparser-4.10.4.jar" +
+                File.pathSeparator + "/usr/share/java/slf4j-api.jar" +
+                File.pathSeparator + "/usr/share/java/osgi.compendium.jar" +
+                File.pathSeparator + "/usr/share/java/jts-core.jar" +
                 File.pathSeparator + javaToolsJar;
         FileList files;
         if (clientOnly) {
